platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -209,7 +210,9 @@ optional<std::pair<Response, uint64_t>> OfflineDatabase::getResource(const Resou
     accessedStmt->bind(1, util::now());
     accessedStmt->bind(2, resource.url);
     accessedStmt->run();
-
+    
+     */
+  
```

Comment:
dont set the accessed timestamp for resources. We turned off setting this for tiles awhile ago, and it is the only place that the priority thread writes to the database.